app/Policies/PlatformPolicy.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Policies;
+
+use App\Models\Platform;
+use App\Models\Role;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class PlatformPolicy
+{
+    use HandlesAuthorization;
+
+    public function manage(User $user): bool
+    {
+        return $user->hasAnyRole([
+            Role::RELEASE_MANAGER,
+        ]);
+    }
+
+    public function viewAny(?User $user): bool
+    {
+        if (!$user) {
+            return false;
+        }
+
+        return $user->hasAnyRole([
+            Role::ROOT,
+            Role::RELEASE_MANAGER,
+        ]);
+    }
+
+    public function view(User $user, Platform $platform): bool
+    {
+        return $user->hasAnyRole([
+            Role::RELEASE_MANAGER,
+        ]);
+    }
+
+    public function create(User $user): bool
+    {
+        return $user->hasAnyRole([
+            Role::RELEASE_MANAGER,
+        ]);
+    }
+
+    public function update(User $user, Platform $platform): bool
+    {
+        return $user->hasAnyRole([
+            Role::RELEASE_MANAGER,
+        ]);
+    }
+
+    public function delete(User $user, Platform $platform): bool
+    {
+        return false;
```

Comment:
Can still delete platforms using Bulk actions

Reply:
Weird, I'm surprised Filament didn't pick up on the policy here. Updated in latest to remove the bulk action from the resource.